client/src/types/boost.ts

```diff
@@ -1,10 +1,115 @@
-export type BoostConfigType = 'powerPlan' | 'rider' | 'bike' | 'track' | 'all';
+import {
+  Number,
+  String,
+  Array,
+  Record,
+  Union,
+  Runtype,
+  Static,
+} from 'runtypes';
+
+const RiderT = Record({
```

Comment:
I would like to have a convention where runtime types are denoted with `RT` and actual TS types are denoted with `T` so it's more clear what types we're using. This is because runtime types are values whereas TS types are not, which can result in a lot of confusion when they're mixed.